src/test/java/com/alkemy/ong/common/BaseContactTest.java

```diff
@@ -0,0 +1,45 @@
+package com.alkemy.ong.common;
+
+import com.alkemy.ong.dto.ContactRequestDto;
+import com.alkemy.ong.model.Contact;
+import com.alkemy.ong.repository.ContactRepository;
+
+import java.util.ArrayList;
+import java.util.List;
+
+import org.springframework.boot.test.mock.mockito.MockBean;
+
+public class BaseContactTest extends BaseGeneralTest{
+	
+	@MockBean
+    protected ContactRepository contactRepository;
+	
+    protected ContactRequestDto contactRequest() {
+    	ContactRequestDto contactRequestDto = new ContactRequestDto();
+    	contactRequestDto.setName("Carlos");
+    	contactRequestDto.setEmail("Carlos@hotmail.com");
+    	contactRequestDto.setMessage("mensaje de prueba");
+        return contactRequestDto;
+    }
+
+    protected List<Contact> generateContacts(int count){
```

Comment:
No esta mal, pero creacion de Contact se podria hacer dentro del test correspondiente bajo la annotation @Before y nos evitabamos hacer el for. Pero lo abstrayeron en otra clase, es valido